lib/jsdom/living/postmessage.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+const isValidTargetOrigin = require("../utils").isValidTargetOrigin;
+
+module.exports = function (message, targetOrigin) {
+  let window = this;
+  let event;
+
+  if (!targetOrigin) {
```

Comment:
How about `arguments.length !== 2` with the error message `"'postMessage' requires 2 arguments: 'message' and 'targetOrigin'"`?